etc/vcl_snippets/deliver.vcl

```diff
@@ -17,6 +16,23 @@
         # Remove X-Magento-Vary and HTTPs Vary served to the user
         set resp.http.Vary = regsub(resp.http.Vary, "X-Magento-Vary,Https", "Cookie");
         remove resp.http.X-Magento-Tags;
+
+        # Set headers based on suggestions in https://www.fastly.com/blog/headers-we-want
+        set resp.http.Content-Security-Policy = "default-src 'self'; frame-ancestors 'self'";
```

Comment:
This might be too restricting.  Won't this cause things like Google Analytics to fail?  Do our merchants not commonly use content from other websites?
Don't we need to document how merchants can add to this list for support when using third-party API services?

Reply:
Yes, it seems unlikely that you can predict all third party resources that the magento site uses.

Reply:
I have remove this in the latest commit.